### PR TITLE
feat(ColorMode): correct background-current from useColorMode hook

### PR DESCRIPTION
**Pull request by dreamwasp**
### Overview

adds correct `background-current` to useColorMode hook + adjusts the mobile GlobalHeader to match these [designs](https://www.figma.com/file/40NSFXquATvE19Gm9pnJzI/Catalog-navigation-dropdown?node-id=1080%3A18028) 

### PR Checklist

- [x] Related to JIRA ticket:[ [ABC-123]](https://www.figma.com/file/40NSFXquATvE19Gm9pnJzI/Catalog-navigation-dropdown?node-id=1080%3A18028)
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

* Go to the GlobalHeader page in Storybook
* Nav to "Free User - Default" (the first example on the page)
* When the mobile menu is open, the headerbar should be beige (like so)
* 
image

* Go to each of the respective PR envs + follow each of the testing instructions there 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/32783)  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/299)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-299)   |

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.8.1-alpha.f7566a.0<br/>@codecademy/gamut-illustrations@0.32.1-alpha.f7566a.0<br/>@codecademy/gamut-kit@0.6.208-alpha.f7566a.0<br/>@codecademy/gamut-labs@33.5.4-alpha.f7566a.0<br/>@codecademy/gamut-patterns@0.8.14-alpha.f7566a.0<br/>@codecademy/gamut-styles@14.0.5-alpha.f7566a.0<br/>@codecademy/gamut-tests@4.0.13-alpha.f7566a.0<br/>@codecademy/gamut@50.6.7-alpha.f7566a.0<br/>@codecademy/markdown-overrides@0.9.11-alpha.f7566a.0<br/>@codecademy/monaco@0.6.89-alpha.f7566a.0<br/>@codecademy/styleguide@61.5.15-alpha.f7566a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6335b4384b5f19433f76aacc--gamut-preview.netlify.app>https://6335b4384b5f19433f76aacc--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6335b4384b5f19433f76aacc>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.8.1-alpha.22d3ab.0<br/>@codecademy/gamut-illustrations@0.32.1-alpha.22d3ab.0<br/>@codecademy/gamut-kit@0.6.208-alpha.22d3ab.0<br/>@codecademy/gamut-labs@33.5.4-alpha.22d3ab.0<br/>@codecademy/gamut-patterns@0.8.14-alpha.22d3ab.0<br/>@codecademy/gamut-styles@14.0.5-alpha.22d3ab.0<br/>@codecademy/gamut-tests@4.0.13-alpha.22d3ab.0<br/>@codecademy/gamut@50.6.7-alpha.22d3ab.0<br/>@codecademy/markdown-overrides@0.9.11-alpha.22d3ab.0<br/>@codecademy/monaco@0.6.89-alpha.22d3ab.0<br/>@codecademy/styleguide@61.5.15-alpha.22d3ab.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6335c18a7041fe005ca8a015--gamut-preview.netlify.app>https://6335c18a7041fe005ca8a015--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6335c18a7041fe005ca8a015>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.8.1-alpha.af4098.0<br/>@codecademy/gamut-illustrations@0.32.1-alpha.af4098.0<br/>@codecademy/gamut-kit@0.6.208-alpha.af4098.0<br/>@codecademy/gamut-labs@33.5.4-alpha.af4098.0<br/>@codecademy/gamut-patterns@0.8.14-alpha.af4098.0<br/>@codecademy/gamut-styles@14.0.5-alpha.af4098.0<br/>@codecademy/gamut-tests@4.0.13-alpha.af4098.0<br/>@codecademy/gamut@50.6.7-alpha.af4098.0<br/>@codecademy/markdown-overrides@0.9.11-alpha.af4098.0<br/>@codecademy/monaco@0.6.89-alpha.af4098.0<br/>@codecademy/styleguide@61.5.15-alpha.af4098.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6335c49b4bed440a41142904--gamut-preview.netlify.app>https://6335c49b4bed440a41142904--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6335c49b4bed440a41142904>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@7.8.1-alpha.d2a31c.0<br/>@codecademy/gamut-illustrations@0.32.1-alpha.d2a31c.0<br/>@codecademy/gamut-kit@0.6.208-alpha.d2a31c.0<br/>@codecademy/gamut-labs@33.5.4-alpha.d2a31c.0<br/>@codecademy/gamut-patterns@0.8.14-alpha.d2a31c.0<br/>@codecademy/gamut-styles@14.0.5-alpha.d2a31c.0<br/>@codecademy/gamut-tests@4.0.13-alpha.d2a31c.0<br/>@codecademy/gamut@50.6.7-alpha.d2a31c.0<br/>@codecademy/markdown-overrides@0.9.11-alpha.d2a31c.0<br/>@codecademy/monaco@0.6.89-alpha.d2a31c.0<br/>@codecademy/styleguide@61.5.15-alpha.d2a31c.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://633afda30343dc086eebeb90--gamut-preview.netlify.app>https://633afda30343dc086eebeb90--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/633afda30343dc086eebeb90>Deploy Logs</a></p>

**Review by jakemhiller — APPROVED** [review]
LGTM!

File: packages/gamut-labs/src/AppHeaderMobile/index.tsx [hunk 1]
```diff
@@ -78,6 +78,8 @@ export const AppHeaderMobile: React.FC<AppHeaderMobileProps> = ({
   setOpenCrossDeviceItemId,
   crossDeviceBookmarkParts,
 }) => {
+  const [mode, , modes] = useColorModes();
```
Comment (zippyzow) [inline packages/gamut-labs/src/AppHeaderMobile/index.tsx:81]: is the extra comma intentional?

**Review by zippyzow — COMMENTED** [review]
Thanks so much for tackling this. Have two questions (one is a comment in the code) and the other is should we be seeing the beige color in storybook on desktop as well?
Screen Shot 2022-10-03 at 12 33 49 PM

**Comment by dreamwasp** [comment]
> Thanks so much for tackling this. Have two questions (one is a comment in the code) and the other is should we be seeing the beige color in storybook on desktop as well? Screen Shot 2022-10-03 at 12 33 49 PM

i wrapped this example in a beige background so it should be seen here!

File: packages/gamut-labs/src/AppHeaderMobile/index.tsx (hunk 1 above)
Comment (dreamwasp) [reply packages/gamut-labs/src/AppHeaderMobile/index.tsx:81]: yes -- the values here are returned in a tuple but we aren't using the second value here

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://633b3a0318b80c29384d1d1a--gamut-preview.netlify.app>https://633b3a0318b80c29384d1d1a--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/633b3a0318b80c29384d1d1a>Deploy Logs</a></p>

File: packages/gamut-labs/src/AppHeaderMobile/index.tsx (hunk 1 above)
Comment (zippyzow) [reply packages/gamut-labs/src/AppHeaderMobile/index.tsx:81]: cool TIL

**Review by zippyzow — APPROVED** [review]

**Pull request merged**